traffic_router/core/src/main/java/com/comcast/cdn/traffic_control/traffic_router/core/dns/ZoneManager.java

```diff
@@ -285,38 +336,43 @@ public StatTracker getStatTracker() {
 		return createZoneCache(cacheType, CacheBuilderSpec.parse(""));
 	}
 
-	private static LoadingCache<ZoneKey, Zone> createZoneCache(final ZoneCacheType cacheType, final CacheBuilderSpec spec) {
+	private static LoadingCache<ZoneKey, Zone> createZoneCache(final ZoneCacheType cacheType,
+	                                                           final CacheBuilderSpec spec) {
 		final RemovalListener<ZoneKey, Zone> removalListener = new RemovalListener<ZoneKey, Zone>() {
 			public void onRemoval(final RemovalNotification<ZoneKey, Zone> removal) {
-				LOGGER.debug(cacheType + " " + removal.getKey().getClass().getSimpleName() + " " + removal.getKey().getName() + " evicted from cache: " + removal.getCause());
+				LOGGER.debug(cacheType + " " + removal.getKey().getClass().getSimpleName() + " " + removal.getKey()
+						.getName() + " evicted from cache: " + removal.getCause());
 			}
 		};
 
 		return CacheBuilder.from(spec).recordStats().removalListener(removalListener).build(
-			new CacheLoader<ZoneKey, Zone>() {
-				final boolean writeZone = (cacheType == ZoneCacheType.STATIC) ? true : false;
+				new CacheLoader<ZoneKey, Zone>() {
```

Comment:
in the future please refrain from doing massive whitespace changes like this; it's difficult to review with other non-whitespace changes. If you'd like to change/improve so much formatting, please do so in a separate, formatting-only PR.